src/databricks/azext_databricks/_params.py

```diff
@@ -30,6 +31,11 @@ def load_arguments(self, _):
     with self.argument_context('databricks workspace update') as c:
         c.argument('workspace_name', options_list=['--name', '-n'], id_part='name', help='The name of the workspace.')
         c.argument('tags', tags_type)
+        c.extra('assign_identity', action='store_true', help='Enable the Managed Identity for managed storage account.')
+        c.argument('encryption_key_source', options_list=['--key-source'], arg_group='Encryption', arg_type=get_enum_type(['Default', 'Microsoft.Keyvault']), validator=validate_encryption_values, help='The encryption key source (provider).')

```

Comment:
worth to sync with PM to see is it necessary to define consistence on parms name now